bin/build.js

```diff
@@ -41,19 +43,27 @@ else {
       port: SERVE_PORT,
     })
     .then(async ({ port }) => {
+      const buildDirectory = join(process.cwd(), BUILD_DIRECTORY);
+      const files = readdirSync(buildDirectory);
+
       // Log all served files for easy reference
       console.table(
-        ENTRY_POINTS.map((path) => {
-          const file = path.replace('src/', '').replace('.ts', '.js');
-          const location = `http://localhost:${port}/${file}`;
-          const script = `<script defer src="${location}"></script>`;
-
-          return {
-            'Built File': file,
-            'File Location': location,
-            'Script Suggestion': script,
-          };
-        })
+        files
+          .map((file) => {
+            if (file.endsWith('map')) return;
+
+            const location = `http://localhost:${port}/${file}`;
+            const tag = file.endsWith('css')
+              ? `<link href="${location}" rel="stylesheet" type="text/css"/>`
+              : `<script defer src="${location}"></script>`;
```

Comment:
so by default all scripts are set to be executed after the page has finished loading? or we let the user decide

Reply:
The `<script>` tag is just a suggestion. If the user is advanced enough to know about `async` imports, they'll know how to change it if they require.